index.md

```diff
@@ -46,25 +48,42 @@ title: "Catalog of IDR images formatted as OME-NGFF"
         <tr>
             <td>{{ rec["OME-NGFF version"] }}</td>
             <td>
-                <a href="http://hms-dbmi.github.io/vizarr/?source={{ rec[s3key] }}">
+                <a target="_blank" href="http://hms-dbmi.github.io/vizarr/?source={{ rec[s3key] }}">
                     <img
                         alt="IDR thumbnail for image:{{image_id}}"
                         style="margin:0"
-                        src="https://idr.openmicroscopy.org/webclient/render_thumbnail/{{image_id}}/"/>
+                        {% if rec["Thumb ID"] %}
```

Comment:
`Thumb ID` is a bit cryptic and especially as this is used a `image:` above. Maybe this should be called `Representative Image ID`?

And although it might lead to duplication, should we consider filling this ID for every row of the table and remove the parsing logic?

Reply:
IDR has a "Study Example Image" tag to identify the example Image for each *Study*. I guess this isn't quite the same. Happy to go with `Representative Image ID`.

Reply:
Added `Representative Image ID` for all rows, but we still need the parsing of `ID` from zarr endpoint to get the `plate ID` for linking to IDR.
Probably HCS data should be in a different table, then we can have custom tables for Images vv Plates and make it easier to find them etc. This PR or a follow-up?

Reply:
Yes - this Fixes #3.

Reply:
In my mind, maintaining 1 table vs several (per specification?) is still an open question. I am still weighing the pros and the cons and this is why I captured #3 as an issue.

If you think we might evolve towards splitting them, how easy would it be to produce a prototype of the 2 table version that we could review and compare?